StudyfiedBackend/Controllers/Quiz/QuizHelper.cs

```diff
@@ -100,14 +101,30 @@ public static bool isValidQuiz(Quiz quiz, int numberOfQuestion)
 
             bool isValidQuiz = true;
 
-            bool isValidNumberOfQuestions = quiz.questionAnswerPairs.Keys
-                .Select(question => question.question != null && question.question != "")
-                .Count() == numberOfQuestion;
+            int numOfQuizQuestions = 0;
+
+            foreach (string question in quiz.questionAnswerPairs.Keys)
+            {
+                if (question == null)
+                {
+                    quiz.questionAnswerPairs.Remove(question);
+                    break;
+                }
+                if (question.Length < 5)
```

Comment:
this doesn't make sense, each question a string, you can't know if the question's length is less than 5 caracters

and the questions length doesnt matter in the first place, the shape of the question (length, nullability, etc) is what matters